src/app/Components/LanguageAvatar.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+  export let name: string;
+  export let selectedLanguage: string = "";
+
+  export let isSelected: boolean = name === selectedLanguage;
+  export let className: string = "avatar";
+  if (isSelected) {
+    className = className + " selected";
+  }
+
+  /**
+   * Handles the click when a language is selected from the sidebar.
+   * Should change the selected language variable and change content in the Languages component.
+   * TODO: Replace stub
+   *
+   * @param {string} languageName - A the name of the language selected.
+   * @return {void}
+   */
+  const handleClick = (languageName: string): void => {
+    console.log(languageName + " button selected");
+  };
+</script>
+
+<style>
+  .avatar {
+    width: 50px;
+    height: 50px;
+    display: flex;
+    background-color: var(--white);
+    align-items: center;
+    justify-content: center;
+    border-radius: 100%;
+    margin-top: 40px;
+    font-family: Cabin, sans-serif;
+    font-size: 20px;
+    box-sizing: border-box;
+    transition: border-color 0.2s ease;
+    border: 3px solid var(--white);
+  }
+
+  .avatar:hover {
+    cursor: pointer;
+    box-shadow: 0px 10px 20px rgba(0, 0, 0, 0.1);
+    border: 3px solid var(--primary-blue);
+  }
+
+  .selected {
+    background-color: var(--primary-blue);
+    border: 0px;
+    color: var(--white);
+    box-shadow: 0px 10px 20px rgba(0, 0, 0, 0.1);
+  }
+</style>
+
+<div
+  class={className}
+  on:click={() => {
+    handleClick(name);
+  }}>
+  {name.substring(0, 1)}
```

Comment:
I am **not** going to request changes for this one change, but... since we're dealing with the worlds' languages, not all of them can use `name.substring(0, 1)` to get the first _abstract character_. There are many reasons for this, but it includes _grapheme clusters_  and in some dumb cases, _surrogate pairs_. [I have an entire presentation devoted to these two issues](https://eddieantonio.ca/unicode-es6/#/).